src/dashboard/install/README.md

```diff
@@ -34,7 +49,48 @@ Please be aware that Archivematica supports different types of distributions
 (Ubuntu/CentOS packages, Ansible or Docker images) and they may override some
 of these settings or provide values to mandatory fields.
 
-This is the full list of environment strings supported:
+## Configuration files
+
+The Dashboard will look for a configuration file in one location:
+-`/etc/archivematica/archivematicaCommon/dbsettings`
```

Comment:
This list isn't formatted properly. I can't tell if it's the lack of indentation or the fact that it's not separated from the paragraph like we do in other places in this document.